docs/use.md

```
@@ -0,0 +1,296 @@
+A **package family** is the set of installed variants that share the same package name. These variants can differ in several ways.
+
+| Variant Type | Example | Description |
+|--------------|---------|-------------|
+| **Version** | `python@3.11` vs `python@3.12` | Different versions of the same package |
+| **Implementation** | `cat` (GNU) vs `cat` (BusyBox) | Different implementations providing the same binary |
+| **Repository** | `neovim` from `main` vs `testing` | Same package from different repositories |
+| **Package ID** | `coreutils` vs `coreutils-ucr` | Different package IDs providing similar functionality |
+
+### Example Package Families
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)